### ISSUE TITLE
Fail to run docker-compose up -d

### ISSUE THREAD

**Issue by bj0rge** [mealie-recipes/mealie#700]
**Describe the bug**
Impossible to run the docker env

**Steps To Reproduce**
1. Run `docker-compose up -d`
2. Wait for it to fail
3. Try not to cry
4. Cry a lot

**Expected behavior**
Build the container

**Actual Behavior**
Failing to build the container on poetry install step

**Device Information (please complete the following information):**
 - OS: Debian 10 buster
 - Deployment: docker-compose

**Additional context**

It looks like [the latest `setuptools` version doesn't support `use_2to3` anymore](https://github.com/pypa/setuptools/issues/2781#issuecomment-917185367), while mealie still relies on it
I don't know python well enough to be able to pin setuptools version and go forward with the install of mealie, do you plan to fix that dependencie issue?

<details>
<summary>error log</summary>

```
Step 18/50 : RUN poetry install -E pgsql --no-dev
 ---> Running in 25edce251b70
Creating virtualenv mealie in /opt/pysetup/.venv
Installing dependencies from lock file

Package operations: 70 installs, 0 updates, 0 removals

  • Installing six (1.16.0)
  • Installing certifi (2021.5.30)
  • Installing charset-normalizer (2.0.3)
  • Installing idna (3.2)
  • Installing isodate (0.6.0)
  • Installing pyparsing (2.4.7)
  • Installing soupsieve (2.2.1)
  • Installing urllib3 (1.26.6)
  • Installing webencodings (0.5.1)
  • Installing beautifulsoup4 (4.9.3)
  • Installing html5lib (1.1)
  • Installing lxml (4.6.2)
  • Installing pycparser (2.20)
  • Installing rdflib (6.0.0)
  • Installing requests (2.26.0)
  • Installing cffi (1.14.6)
  • Installing decorator (5.0.9)
  • Installing html-text (0.5.2)
  • Installing jstyleson (0.0.2)
  • Installing mf2py (1.1.2)
  • Installing oauthlib (3.1.1)
  • Installing pyasn1 (0.4.8)
  • Installing pyrdfa3 (3.5.3)
  • Installing pytz (2021.1)
  • Installing rdflib-jsonld (0.5.0)
  • Installing typing-extensions (3.10.0.0)
  • Installing w3lib (1.22.0)

  EnvCommandError

  Command ['/opt/pysetup/.venv/bin/pip', 'install', '--no-deps', '/root/.cache/pypoetry/artifacts/1e/7b/07/d24e7035613a90a65ac7522d01ab70a7e46f490832fa283b50899a6848/rdflib-jsonld-0.5.0.tar.gz'] errored with the following return code 1, and output: 
  Processing /root/.cache/pypoetry/artifacts/1e/7b/07/d24e7035613a90a65ac7522d01ab70a7e46f490832fa283b50899a6848/rdflib-jsonld-0.5.0.tar.gz
      ERROR: Command errored out with exit status 1:
       command: /opt/pysetup/.venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-qzfz7ajc/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-qzfz7ajc/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-fkmcjwyz
           cwd: /tmp/pip-req-build-qzfz7ajc/
      Complete output (1 lines):
      error in rdflib-jsonld setup command: use_2to3 is invalid.
      ----------------------------------------
  WARNING: Discarding file:///root/.cache/pypoetry/artifacts/1e/7b/07/d24e7035613a90a65ac7522d01ab70a7e46f490832fa283b50899a6848/rdflib-jsonld-0.5.0.tar.gz. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  

  at /opt/poetry/venv/lib/python3.9/site-packages/poetry/utils/env.py:1075 in _run
      1071│                 output = subprocess.check_output(
      1072│                     cmd, stderr=subprocess.STDOUT, **kwargs
      1073│                 )
      1074│         except CalledProcessError as e:
    → 1075│             raise EnvCommandError(e, input=input_)
      1076│ 
      1077│         return decode(output)
      1078│ 
      1079│     def execute(self, bin, *args, **kwargs):

The command '/bin/sh -c poetry install -E pgsql --no-dev' returned a non-zero code: 1
ERROR: Service 'mealie' failed to build : Build failed
```
</details>

<details>
<summary>docker-compose.yml content</summary>

```
version: "3.1"
services:
  mealie:
    build:
      context: ./
      target: production
      dockerfile: Dockerfile
    container_name: mealie
    restart: always
    depends_on:
      - "postgres"
    ports:
      - 9090:80
    environment:
      DB_ENGINE: postgres # Optional: 'sqlite', 'postgres'
      POSTGRES_USER: mealie
      POSTGRES_PASSWORD: mydbpwd
      POSTGRES_SERVER: postgres
      POSTGRES_PORT: 5432
      POSTGRES_DB: mealie
      # WORKERS_PER_CORE: 0.5
      # MAX_WORKERS: 8
      WEB_CONCURRENCY: 2
  postgres:
    container_name: postgres
    image: postgres
    restart: always
    environment:
      POSTGRES_PASSWORD: mydbpwd
      POSTGRES_USER: mealie
```
</details>

Thanks a lot guys 💙

**Comment by hay-kot** [comment]
Thanks for reporting, I wasn't aware of this. I think we caught it on the development side just by updating so it should be fixed in the next release (planned for end of week at latest). That said...

Is there a reason you're trying to build instead of following the installation instructions and pulling an already created container? I'm fairly certain you should be able to deploy the docker-compose from the documentation without issue. Try that, if you're still having issues please report back.

**Comment by hay-kot** [comment]
This should be resolved in v0.5.3. Please reopen if you're still experiencing the issue after upgrading.
